modules/cluster-logging-deploy-console.adoc

```diff
@@ -0,0 +1,220 @@
+
+. Create a cluster logging instance:
+
+.. Switch to the *Administration* -> *Custom Resource Definitions* page.
+
+.. On the *Custom Resource Definitions* page, click *ClusterLogging*.
+
+.. On the *Custom Resource Definition Overview* page, select *View Instances* from the *Actions* menu.
+
+.. On the *ClusterLoggings* page, click *Create ClusterLogging*.
++
+You might have to refresh the page to load the data.
+
+.. In the YAML field, replace the code with the following:
++
+[NOTE]
+====
+This default cluster logging configuration should support a wide array of environments. Review the topics on tuning and
+configuring the cluster logging components for information on modifications you can make to your cluster logging cluster.
+====
++
+[source,yaml]
+----
+apiVersion: "logging.openshift.io/v1"
+kind: "ClusterLogging"
+metadata:
+  name: "instance" <1>
+  namespace: "openshift-logging"
+spec:
+  managementState: "Managed" <2>
+  logStore:
+    type: "elasticsearch" <3>
+    retentionPolicy: <4>
+      application:
+        maxAge: 7d
+      infra:
+        maxAge: 0d
+      audit:
+        maxAge: 0d
+    elasticsearch:
+      nodeCount: 3 <5>
+      storage:
+        storageClassName: gp2 <6>
+        size: "200Gi"
+      redundancyPolicy: "SingleRedundancy"
+      nodeSelector:
+        node-role.kubernetes.io/worker: ""
+      resources: <7>
+        request:
+          memory: 8G
+  visualization:
+    type: "kibana" <8>
+    kibana:
+      replicas: 1
+      nodeSelector:
+        node-role.kubernetes.io/worker: ""
+  curation:
+    type: "curator"
+    curator:
+      schedule: "30 3 * * *" <9>
+      nodeSelector:
+        node-role.kubernetes.io/worker: ""
+  collection:
+    logs:
+      type: "fluentd" <10>
+      fluentd: {}
+      nodeSelector:
+        node-role.kubernetes.io/worker: ""
+----
+
+<1> The name must be `instance`.
+<2> The cluster logging management state. In some cases, if you change the cluster logging defaults, you must set this to `Unmanaged`.
+However, an unmanaged deployment does not receive updates until the cluster logging is placed back into a managed state.
+<3> Settings for configuring Elasticsearch. Using the CR, you can configure shard replication policy and persistent storage.
+<4> You must specify the length of time that Elasticsearch should retain each log source. Enter an integer for the number of days, for example, `7d` for seven days. The `application` `maxAge` can be less than or equal to `7d`. The `infra` `maxAge` and `audit` `maxAge` must be `0d`, and this value *must be explicitly set* to override the default. Logs older than the `maxAge` are deleted. You must specify a retention policy for each log source or the Elasticsearch indices will not be created for that source.
+<5> Specify the number of Elasticsearch nodes. See the note that follows this list.
+<6> You must enter `gp2` for the Elasticsearch `storageClassname`.
```

Comment:
This is only for AWS. Other environments (like GCP) have a different configuration here.